commit 11d797ef5defb75362e40cfda24dc88c4320d22f
Author: Emil Edholm <emil@edholm.it>
Date:   Wed Feb 10 20:18:28 2016 +0100

    Remove min cache time for rsstorrents provider

diff --git a/sickbeard/providers/rsstorrent.py b/sickbeard/providers/rsstorrent.py
index a901ee0..d7ea50a 100644
--- a/sickbeard/providers/rsstorrent.py
+++ b/sickbeard/providers/rsstorrent.py
@@ -40,7 +40,7 @@ class TorrentRssProvider(TorrentProvider):  # pylint: disable=too-many-instance-
 
         TorrentProvider.__init__(self, name)
 
-        self.cache = TorrentRssCache(self, min_time=15)
+        self.cache = TorrentRssCache(self, min_time=0)
         self.urls = {'base_url': re.sub(r'/$', '', url)}
         self.url = self.urls['base_url']
         self.ratio = None
diff --git a/sickbeard/tvcache.py b/sickbeard/tvcache.py
index ddfc9b9..4cec83d 100644
--- a/sickbeard/tvcache.py
+++ b/sickbeard/tvcache.py
@@ -77,7 +77,7 @@ class TVCache(object):
         self.provider = provider
         self.providerID = self.provider.get_id()
         self.providerDB = None
-        self.minTime = kwargs.pop(u'min_time', 10)
+        self.minTime = kwargs.pop(u'min_time', 0)
         self.search_params = kwargs.pop(u'search_params', dict(RSS=['']))
 
     def _getDB(self):
